app/templates/index.html

```diff
@@ -78,6 +78,13 @@ <h3 class="heading-xmedium">Buyers</h3>
       </p>
   {% endif %}
 
+  {% if current_user.has_any_role('admin-ccs-category', 'admin-framework-manager', 'admin-ccs-sourcing') %}
```

Comment:
We have comprehensive tests for what each role can see on the admin home page (`test_index_page.py`) - let's add in coverage for this link.